app.cfg.example

```diff
@@ -306,19 +306,78 @@ signing =
+        "repo_targets": ["eessi.io-2023.06-compat","eessi.io-2023.06-software"]
+    },
+    # This is a CPU partition. We specify an explicit "accel": "None" property. Thus, this partition will only be
+    # used if the bot build command does NOT contain an accel argument, e.g.
+    # bot: build instance:xyz repo:eessi.io-2023.06-software arch:zen4
+    # will cause the event_filter to mark this virtual partition as a valid match, and will use it to start building
+    # for zen4.
+    # When invoking the bot with an accelerator command, such as 
+    # bot: build instance:xyz repo:eessi.io-2023.06-software arch:zen4 accel:nvidia/cc90
+    # the event_filter will NOT mark this virtual partition as a valid match. This is intentional, as this particular
+    # (example) cluster has a native zen4+cc90 partition (gpu_h100) and we want this command to trigger a native build
+    # on that partition, rather than cross-compiling on this cpu_zen4 partition.
+    # One could still allow cross-compilation for other accelerator architectures, e.g. cc70 and cc80 by defining
+    # "accel": ["nvidia/cc70", "nvidia/cc80"]
+    "cpu_zen4": {
+        "os": "linux",
+        "cpu_subdir": "x86_64/amd/zen4",
+        "accel": ["None"],
+        "slurm_params": "-p genoa --nodes 1 --ntasks-per-node 24 --cpus-per-task 1",
+        "repo_targets": ["eessi.io-2023.06-compat","eessi.io-2023.06-software"]
+    },
+    # This is a GPU partition. We specify an explicit "accel" property. Thus, only if the bot build command
+    # specifies that explicit accelerator in combination with the relevant CPU type,
+    # bot: build instance:xyz repo:eessi.io-2023.06-software arch:icelake accel:nvidia/cc80
+    # will a build be triggered on this partition
+    # If you want to use this partition also for CPU only builds, you can alter the "accel" property to
+    # "accel": ["None", "nvidia/cc80"]
+    "gpu_a100": {
+        "os": "linux",
+        "cpu_subdir": "x86_64/intel/icelake",
+        "accel": ["nvidia/cc80"],
+        "slurm_params": "-p gpu_a100 --nodes 1 --tasks-per-node 18 --cpus-per-task 1 --gpus-per-node 1",
+        "repo_targets": ["eessi.io-2023.06-compat","eessi.io-2023.06-software"]
+    },
+    # This is a GPU partition. We specify an explicit "accel" property. Thus, only if the bot build command
+    # specifies that explicit accelerator in combination with the relevant CPU type,
+    # bot: build instance:xyz repo:eessi.io-2023.06-software arch:zen4 accel:nvidia/cc90
+    # will a build be triggered on this partition
+    # If you want to use this partition also for cross-compiling for cc70 and cc80 architectures, you can alter
+    # the "accel" property to 
+    # "accel": ["nvidia/cc70", "nvidia/cc80", "nvidia/cc90"]
+    # Note that setting:
+    # "accel": ["None", "nvidia/cc90"]
+    # is invalid here, since it would lead to both the cpu_zen4 and the gpu_h100 partitions matching the build command
```

Comment:
"Invalid" in the sense the bot checks and refuses to even start or "invalid" in the sense it is not recommended because it's ambiguous?

Reply:
Invalid in the sense it will lead to an error (the bot will fail on prepare the job dir for the gpu_h100 partition because that job dir was already prepared for the same job on cpu_zen4).

Reply:
Which would never happen if partitions have unique names and a user targets a partition by its name. Would also be easier to understand what is going on.